db/migration/1711580214803-RemoveAvailableEntitiesFromChartConfigs.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class RemoveAvailableEntitiesFromChartConfigs1711580214803
+    implements MigrationInterface
+{
+    // Removes `$.data.availableEntities` from chart configs
+    // In doing that, we can also remove the whole `$.data` object, because it is not used anymore
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const configColumns = [
+            { table: "charts", column: "config" },
+            { table: "chart_revisions", column: "config" },
+            { table: "suggested_chart_revisions", column: "suggestedConfig" },
+            { table: "variables", column: "grapherConfigAdmin" },
+            { table: "variables", column: "grapherConfigETL" },
+        ]
+
+        for (const { table, column } of configColumns) {
+            await queryRunner.query(
+                `-- sql
+                UPDATE ??
+                SET ?? = JSON_REMOVE(??, "$.data")
+                WHERE JSON_CONTAINS_PATH(??, "one", "$.data")
+            `,
+                [table, column, column, column]
+            )
+
+            // Update schema version to 004
+            await queryRunner.query(
+                `--sql
```

Comment:
```suggestion
                `-- sql
```
Mysql is stupid and trips over comments if they don't start with a space. The dbtests are broken because of this.